notebook/agentchat_logging.ipynb

```diff
@@ -287,6 +291,82 @@
     "    + str(round(session_cost, 4))\n",
     ")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log data in File mode\n",
+    "\n",
+    "By default, the log type is set to `sqlite` as shown above, but we introduced a new parameter for the `autogen.runtime_logging.start()`\n",
+    "\n",
+    "the `logger_type = \"file\"` will start to log data in the File mode."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Logging session ID: ed493ebf-d78e-49f0-b832-69557276d557\n",
+      "\u001b[33muser_proxy\u001b[0m (to assistant):\n",
+      "\n",
+      "What is the height of the Eiffel Tower? Only respond with the answer and terminate\n",
+      "\n",
+      "--------------------------------------------------------------------------------\n",
+      "\u001b[33massistant\u001b[0m (to user_proxy):\n",
+      "\n",
+      "The height of the Eiffel Tower is 330 meters.\n",
+      "TERMINATE\n",
+      "\n",
+      "--------------------------------------------------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "import autogen\n",
+    "from autogen import AssistantAgent, UserProxyAgent\n",
+    "\n",
+    "# Setup API key. Add your own API key to config file or environment variable\n",
+    "llm_config = {\n",
+    "    \"config_list\": autogen.config_list_from_json(\n",
+    "        env_or_file=\"OAI_CONFIG_LIST\",\n",
+    "    ),\n",
+    "    \"temperature\": 0.9,\n",
+    "}\n",
+    "\n",
+    "# Start logging with logger_type\n",
+    "logging_session_id = autogen.runtime_logging.start(logger_type=\"file\")\n",
```

Comment:
As we are moving toward using user-supplied logger in `start`, we can create the FileLogger object and use the `logger` argument of `start`. Same with SqliteLogger

Reply:
Would be useful to specify the file location when you create the FileLogger.

Reply:
Sure, let me use a file location instead of default.